cartography/data/jobs/cleanup/aws_dns_cleanup.json

```diff
@@ -6,53 +6,59 @@
       "iterationsize": 100
     },
     {
-      "query": "MATCH (n:NameServer)-[:NAMESERVER]->(:DNSZone)<-[:RESOURCE]-(:AWSAccount{id: {AWS_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
+      "query": "MATCH (n:NameServer)-[:NAMESERVER]->(:DNSZone)<-[:RESOURCE]-(:AWSAccount{id: {AWS_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n)",
```

Comment:
`return COUNT(*) as TotalCompleted` is no longer needed as of https://github.com/lyft/cartography/pull/680 where we emit the number of assets affected by a statement.

Reply:
I had a couple questions.
1) I didn't quite catch why this is being removed. Can you explain? :sweat_smile:
2) Does this need to be done for all cleanup files?

Reply:
Alright!